test/test_cases/run_1d_ekman_column.jl

```diff
@@ -1,7 +1,14 @@
-include("initial_conditions/ekman_column_1d.jl")
+using CLIMAParameters
+using ClimaCore: Geometry
+using ClimaAtmos.Utils.InitialConditions: init_1d_ekman_column
+using ClimaAtmos.Domains
+using ClimaAtmos.BoundaryConditions
+using ClimaAtmos.Models
+using ClimaAtmos.Models.SingleColumnModels
+using ClimaAtmos.Callbacks
+using ClimaAtmos.Simulations
```

Comment:
These packages used to be importing in runtest.jl and can only be imported once. Is there a reason why you want to move it to every test file?

Reply:
Yes, for clarity of the user.